test/e2e/lazy_activation_policy_test.go

```diff
@@ -60,6 +62,16 @@ var _ = ginkgo.Describe("Lazy activation policy testing", func() {
 				ClusterNames: []string{originalCluster},
 			},
 		})
+		policyHigherPriority = testhelper.NewLazyExplicitPriorityPropagationPolicy(namespace, policyHigherPriorityName, []policyv1alpha1.ResourceSelector{
```

Comment:
I feel like using the `NewPropagationPolicy` method and then changing the Preemption value is the way to go.

We do not need to add so many NewPolicy methods.

Reply:
+1

Reply:
+1